openedx/core/djangoapps/user_api/models.py

```diff
@@ -113,3 +113,24 @@ class UserOrgTag(TimeStampedModel):
 
     class Meta(object):
         unique_together = ("user", "org", "key")
+
+
+class InstructorAvailableSections(models.Model):
```

Comment:
Probably from the logic point of view it is better to create new `models.py` script in folder `lms/djangoapps/instructor` and move this class there. Because as I could see the current script `openedx/core/djangoapps/user_api/models.py` contains models for the other goals